### PR TITLE
chore(localfindings): add new filter workflow for local findings

### PR DESCRIPTION
**Pull request by sandor-trombitas**
## Pull Request Submission Checklist
- [x] Follows [CONTRIBUTING](https://github.com/snyk/cli/blob/main/CONTRIBUTING.md) guidelines
- [x] Includes detailed description of changes
- [ ] Contains risk assessment (Low | Medium | High)
- [ ] Highlights breaking API changes (if applicable)
- [ ] Links to automated tests covering new functionality
- [ ] Includes manual testing instructions (if necessary)
- [ ] Updates relevant GitBook documentation (PR link: ___)

## What does this PR do?
- Bumps GAF to add a new filter workflow that filters a local findings model based on severity-threshold

## How should this be manually tested?

```
make build && INTERNAL_SNYK_TRANSFORMATION_WORKFLOW_ENABLED=true ./binary-releases/snyk-macos-arm64 code test --org=<org-slug> . --severity-threshold=medium
```

### CODE REVIEW

File: cliv2/cmd/cliv2/main.go [hunk 1]
```diff
@@ -1,8 +1,6 @@
 package main
 
 // !!! This import needs to be the first import, please do not change this !!!
-import _ "github.com/snyk/go-application-framework/pkg/networking/fips_enable"
```
Comment (j-luong) [inline cliv2/cmd/cliv2/main.go:4]: question: considering the comment above this import, was moving it intentional?

Comment (sandor-trombitas) [reply cliv2/cmd/cliv2/main.go:4]: it was not 😱

File: cliv2/go.sum [hunk 2]
```diff
@@ -764,6 +766,8 @@ github.com/snyk/error-catalog-golang-public v0.0.0-20240809094525-c48d19c27edb h
 github.com/snyk/error-catalog-golang-public v0.0.0-20240809094525-c48d19c27edb/go.mod h1:Ytttq7Pw4vOCu9NtRQaOeDU2dhBYUyNBe6kX4+nIIQ4=
 github.com/snyk/go-application-framework v0.0.0-20241011135148-71eca49aa231 h1:mLiZHx8m36ySB+KZ9x1OR+VR84YJuOXv0/9zYsPkReU=
 github.com/snyk/go-application-framework v0.0.0-20241011135148-71eca49aa231/go.mod h1:LeMsRM1FxIfO/8QpOs9V/dI46ie/RAQl02ulAh6aKys=
+github.com/snyk/go-application-framework v0.0.0-20241025112937-37db20ab7d62 h1:YVJJnwISffl+LUcqOR2P9aCgEsa9DiZXfwGJnHvda1Q=
```
Comment (PeterSchafer) [inline cliv2/go.sum:769]: Seems like you will need to run `go mod tidy`

File: cliv2/cmd/cliv2/main_test.go [hunk 3]
```diff
@@ -487,6 +497,89 @@ func Test_runWorkflowAndProcessData_WithTransformation(t *testing.T) {
...
+
+		for i := range input {
+			mimeType := input[i].GetContentType()
+
+			if strings.HasPrefix(mimeType, content_type.LOCAL_FINDING_MODEL) {
+				localFindingsFound = true
+			}
+		}
+
+		assert.True(t, localFindingsFound)
```
Comment (thisislawatts) [inline cliv2/cmd/cliv2/main_test.go:524]: **issue(blocking):** We want to add in an assertion that validates the data that is received after the filter workflow has run.

**Review by thisislawatts — CHANGES_REQUESTED** [review]

**Pull request merged**